GouMinghao/graspnetAPI
# Moving rc1

### Change
- Query on a simple sequence is added. ("scene_0002" and "036422060215" camera)
- A naive test by generating pred from GT and calculate MGTA(which should be 1.0) has passed which indicate the code is almost OK

### Test Method

- set the MOVING_GRASPNET_ROOT in `tests/test_moving_eval` to that in 6022 server and install the newest package.
- run `python3 run_tests.py`

It should pass all the test in several minutes.

### TODO
- Generate all query grasp poses.
- Add more test cases to verify the correctness.
- Accelerate the run time.
